kDrive/UI/Controller/Files/RootMenuViewController.swift

```diff
@@ -122,8 +122,13 @@ class RootMenuViewController: CustomLargeTitleCollectionViewController, SelectSw
         configureDataSource()
 
         let rootFileUid = File.uid(driveId: driveFileManager.drive.id, fileId: DriveFileManager.constants.rootID)
-        let rootChildren = driveFileManager.getRealm().object(ofType: File.self, forPrimaryKey: rootFileUid)?.children
-        rootChildrenObservationToken = rootChildren?.observe { [weak self] changes in
+        guard let root = driveFileManager.fetchObject(ofType: File.self, forPrimaryKey: rootFileUid),
+              !root.isInvalidated else {
```

Comment:
Same here (https://github.com/Infomaniak/ios-kDrive/pull/1167/files#r1584310026)

Reply:
Double checked.
Since this comes from a throw away realm / Transactionable, it is fine without it.